src/loading.js

```diff
@@ -1,35 +1,43 @@
+/* eslint-disable react/no-multi-comp */
 import React from 'react';
-import map from 'lodash/map';
+import every from 'lodash/every';
 import some from 'lodash/some';
+import omit from 'lodash/omit';
+import constant from 'lodash/constant';
 import displayName from './displayName';
-import _isReady from './isReady';
 
-const _isLoading = ({ readyState }) => {
-  return some(map(readyState, rs => rs.loading));
-};
+export const defaultIsLoading = ({ readyState }) => some(readyState, 'loading');
+
+export const defaultIsReady = ({ readyState }) => every(readyState, 'ready');
+
+export class NoWrapper extends React.Component { render = () => (this.props.children || [])[0] || null; }
```

Comment:
I don't think we can, `loading()` will render
```js
<NoWrapper>
  <Component />
  <Loader />
</NoWrapper>
```
that then results just in `<Component />`